src/main/java/com/mraof/minestuck/block/machine/ComputerBlock.java

```diff
@@ -161,18 +238,37 @@ private void dropItems(Level level, int x, int y, int z, BlockState state)
 			float rx = rand.nextFloat() * 0.8F + 0.1F;
 			float ry = rand.nextFloat() * 0.8F + 0.1F;
 			float rz = rand.nextFloat() * 0.8F + 0.1F;
-			ItemEntity entityItem = new ItemEntity(level, x + rx, y + ry, z + rz, ProgramData.getItem(program));
+			ItemStack diskStack = ProgramData.getItem(program);
+			if(diskStack != null && !diskStack.isEmpty())
```

Comment:
`ProgramData.getItem()` shouldn't be returning null, so the null check here should not be needed.